api/ProfileName/[profileAddress].ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
```

Comment:
we usually dont capitalize directories like ProfileName, and just have it be lower case.